<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BorneFontaineAjouter</class>
 <widget class="QDialog" name="BorneFontaineAjouter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BorneFontaineAjouter</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter une Borne Fontaine</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Identifiant</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Identifiant de la voie publique</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom Topographique</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Longitude</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Latitude</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>340</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ville</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>390</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Arrondissement</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="p_identifiantVoiePublique">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>181</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="p_nomTopographique">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>180</y>
     <width>181</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="p_ville">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>330</y>
     <width>181</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="p_arrondissement">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>380</y>
     <width>181</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ok">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>440</y>
     <width>90</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="p_identifiant">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>181</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>999999</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="p_longitude">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>181</width>
     <height>26</height>
    </rect>
   </property>
   <property name="decimals">
    <number>6</number>
   </property>
   <property name="minimum">
    <double>-180.000000000000000</double>
   </property>
   <property name="maximum">
    <double>180.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="p_latitude">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>280</y>
     <width>181</width>
     <height>26</height>
    </rect>
   </property>
   <property name="decimals">
    <number>6</number>
   </property>
   <property name="minimum">
    <double>-90.000000000000000</double>
   </property>
   <property name="maximum">
    <double>90.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ok</sender>
   <signal>clicked()</signal>
   <receiver>BorneFontaineAjouter</receiver>
   <slot>valider()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>453</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>valider()</slot>
 </slots>
</ui>
